lms/djangoapps/instructor/management/commands/dump_grades.py

```diff
@@ -1,76 +0,0 @@
-#!/usr/bin/python
-"""
```

Comment:
Sure. I'll note that this script relies on code defined in the legacy dashboard that times out for larger courses, and uses a code path we've put behind the flag to hide for courses > 200 students.